modules/nuclide-commons-ui/Atomicon.js

```diff
@@ -35,16 +36,21 @@ const TYPE_TO_CLASSNAME_SUFFIX = {
   variable: 'type-variable',
 };
 
-type AtomiconName = $Keys<typeof TYPE_TO_CLASSNAME_SUFFIX>;
+const ICON_NAME_TO_TYPE = invert(TYPE_TO_ICON_NAME);
```

Comment:
Do we need to bring in loadash just for this?

Reply:
Looks like `lodash` gets pulled in by `rxjs` / `redux` already. @wbinnssmith also added a Babel transform so this essentially gets converted into just `import invert from 'lodash.invert'` as well.